### DISCUSSION TITLE
LOGO importieren

### DISCUSSION THREAD

**Discussion by notfallsicher** [community/community#36386] [Other Feature Feedback, Questions, & Ideas]
Guten Tag

Ich kann mein LOGO nicht importieren, da ich Rechteck habe, aber die Vorlage ist ein Kreis. Was muss ich machen?

**Comment by airtower-luna** [comment]
Ich fürchte du fragst hier an der falschen Stelle, in der GitHub Community geht es hauptsächlich um Git und GitHub. Es klingt als ob du Hilfe mit irgendeiner Software suchst. Falls die Software auf GitHub gehostet ist schau am besten ob im Repository in einer README oder SUPPORT Datei steht wo man Hilfe bekommt. Falls nicht, ist ein Issue oder Discussions-Thread *im Repository für die Software* meist ein guter Anfang.
